.github/workflows/GAP.yml

```diff
@@ -0,0 +1,26 @@
+name: "GAP tests"
+
+# Trigger the workflow on push or pull request
+on:
+  - push
+  - pull_request
+
+jobs:
+  test:
+    name: "GAP ${{ matrix.gap-branch }} on ${{ matrix.os }}"
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+        gap-branch:
+          - master
+          - stable-4.11
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: gap-actions/setup-gap-for-packages@v1
+        with:
+          GAP_PKGS_TO_BUILD: "digraphs genss io orb images datastructures"
+          GAPBRANCH: ${{ matrix.gap-branch }}
+      - uses: gap-actions/run-test-for-packages@v1
```

Comment:
Actually @james-d-mitchell, you _will_ need to do something else.

The action `gap-actions/run-test-for-packages` ultimately runs the file https://github.com/gap-system/pkg-ci-scripts/blob/master/run_tests.sh, which checks the exit code given by GAP when running a particular file.

It runs the file in the variable `GAP_TESTFILE`, if any, but by default it runs `tst/testall.g`. However, the value of `GAP_TESTFILE` is not (yet) passed to the action `gap-actions/run-test-for-packages`. I have opened a PR for this at https://github.com/gap-actions/run-test-for-packages/pull/2; once it is merged, if you wanted, you'd be able to do something like:
```suggestion
      - uses: gap-actions/run-test-for-packages@WHATEVER (probably @master for now)
        with:
          GAP_TESTFILE: "CUSTOM_FILE_PATH"
```
There is a TODO in https://github.com/gap-system/pkg-ci-scripts/blob/master/run_tests.sh suggesting that eventually it will use the value of `TestFile` in the `PackageInfo.g`, if present (which for us would be `tst/teststandard.g`), but that is not yet implemented either.

For now, I think your best bet is to create a new file `tst/testall.g` as I did for Digraphs, and put the tests in there that you want to run. This could be a copy of `tst/teststandard.g`, but you might want to run more stuff too.